### PR TITLE
ES-876652 Added change font table content sample

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Descriptionn:
Added change font table content sample

### CODE REVIEW

File: Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+
+using (FileStream inputStream = new FileStream("Data/Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    // Open the input HTML format document.
+    using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+    {
+        // Access the first section of the document.
+        WSection section = document.Sections[0];
```
Comment (Suriya-Balamurugan) [inline Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs:10]: Remove this line of code.

Also modify input document.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+
+using (FileStream inputStream = new FileStream("Data/Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    // Open the input HTML format document.
+    using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
```
Comment (Suriya-Balamurugan) [inline Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs:7]: Input is DOCX, but xml comments contains HTML.

Change the xml comment properly.

File: Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                // Iterate through each paragraph in the cell.
+                foreach (WParagraph paragraph in cell.Paragraphs)
+                {
+                    // Iterate through the child entities of the paragraph.
+                    foreach (Entity entity in paragraph.ChildEntities)
+                    {
+                        // Check if the child entity is a text range.
+                        if (entity is WTextRange)
+                        {
+                            // Apply character format to change the font to Algerian for the text range.
```
Comment (Suriya-Balamurugan) [inline Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs:26]: Change the xml comment
// Change the font to Algerian for the text range.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs (hunk 2 above)
Comment (VijayadharshiniMathiyalagan) [reply Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs:7]: Addressed the feedback

**Review by Suriya-Balamurugan — APPROVED** [review]
No concerns.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**